src/main/java/com/worksap/nlp/sudachi/MeCabOovProviderPlugin.java

```diff
@@ -167,7 +168,7 @@ <T> void readCharacterProperty(Config.Resource<T> charDef) throws IOException {
         }
     }
 
-    <T> void readOOV(Config.Resource<T> unkDef, Grammar grammar) throws IOException {
+    <T> void readOOV(Config.Resource<T> unkDef, Grammar grammar, String userPosType) throws IOException {
```

Comment:
It would be better to unify whether it is `userPosType` or `userPosMode`.

Reply:
userPosMode it is